sass/responsive/_mobile.scss

```diff
@@ -375,20 +375,25 @@
 
                 .modal-title {
                     max-width: 90%;
-                    float: none;
                 }
 
                 .btn {
                     &.btn-primary {
                         display: block;
                         width: 100%;
                         margin: 10px 0 6px;
-                        float: none;
                     }
                 }
             }
         }
 
+        .more-modal {
```

Comment:
definitely not part of this PR, but I always thought that these very specific changes should be done inside the components SCSS

Reply:
Agreed. The modals are a mess of interconnected and miserable CSS, some of which is from our old version of Bootstrap and some that's specifically to revert the changes from Bootstrap. This is definitely one of those areas that needs a lot of rework or just to be abandoned next time we start reworking modals again.